meilisearch-release-changelog/v1.5.0.md

```diff
@@ -0,0 +1,83 @@
+# v1.5.0 release changelogs
+
+Meilisearch v1.5 introduces improvements in indexing speed and the possibility of triggering snapshots on demand.
+
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Indexing speed improvements
+
+v1.5 improves indexing speed for text-heavy datasets. Datasets with fields containing many words (100 or more) should see a reduction of 5% to 20% to indexing times, with gains proportional to the amount of words in a document.
+
+This might result in minor impact to search result relevancy for queries containing 4 words or more. [Contact us in our GitHub Discussions page](https://github.com/orgs/meilisearch/discussions/704) if you have any issues or unexpected behavior regarding the relevancy.
```

Comment:
```suggestion
This might result in minor impact to search result relevancy for queries containing 4 words or more. [Contact us in our GitHub Discussions page](https://github.com/orgs/meilisearch/discussions/704) if this is significantly affecting your application.
```